src/test/components/MarkerChart.test.js

```diff
@@ -362,17 +358,8 @@ describe('MarkerChart', function() {
       }
 
       function clickOnMenuItem(stringOrRegexp) {
-        fireFullClick(getByText(stringOrRegexp));
-      }
-
-      // Ideally we would simulate hovering on the parent before clicking the
-      // submenu, but this is a bit more work. Maybe later...
-      function clickOnSubMenuItem(parentLookup, submenuLookup) {
-        const parentItem = getByText(parentLookup);
-        // $FlowExpectError Flow thinks parentNode is a Node.
-        const parentNode: HTMLElement = parentItem.parentNode;
-        const submenu = globalGetByText(parentNode, submenuLookup);
-        fireFullClick(submenu);
+        const menuItem = getByRole('menuitem', { name: stringOrRegexp });
```

Comment:
Note: `getByRole` matches also items that are split between different HTML elements (eg: `bla bla <strong>foo bar</strong>`. This isn't the case with `getByText`.

Reply:
TIL about this getByRole behavior. Good to know!